BackEnd/Controllers/user-controller.js

```diff
@@ -1,47 +1,51 @@
 const express = require('express');
 const db = require('../Models/database-model')
+// setting up bcrypt stuff
+const bcrypt = require('bcrypt');
+const saltFactor = 10; 
+
 
 const userController = {};
 
 userController.createUser = (req, res, next) => {
-  // get username and password from request body 
-  const username = req.body.username;
-  const password = req.body.password; 
+    req.locals = {
+        queryData: {},
+    };
+    
+    req.locals.queryData.tableName = 'test3';
```

Comment:
change to users rather than test3. See current dev implementation https://github.com/SharkGoblin/Scratch-Project/blob/dev/BackEnd/Controllers/user-controller.js